README.md

```diff
@@ -20,6 +20,18 @@ $ bundle
 $ bin/rails generate solidus_subscriptions:install
 ```
 
+Or for the new starter frontend:
+
+```shell
+$ bin/rails generate solidus_subscriptions:install --frontend=starter
```

Comment:
Do you mean in the README of in the install script. For the README, yes. For the install script it's a bit more complicated because it would also act on the dummy app used in test, which is still using the "classic" frontend.

Reply:
I meant both, but your comment makes a lot of sense. Let's change the script once we tackle testing.